### DISCUSSION TITLE
openclash的规则

### DISCUSSION THREAD

**Discussion by lost1989** [vernesong/OpenClash#3958] [Q&A]
我登录www.365-213.com这个网站总是提示“您正在尝试在受限的地区使用bet365，我用的是openclash，想自定义这个网站直连，用这个语句没错吧？但是还是不行呢？- DOMAIN-SUFFIX,365-213.com

**Comment by bzzjh** [comment]
- DOMAIN,safebrowsing.urlsec.qq.com,DIRECT
